src/openms/source/METADATA/ExperimentalDesign.cpp

```diff
@@ -599,8 +599,55 @@ namespace OpenMS
       std::set< std::tuple< std::string, unsigned > > path_label_set;
       std::map< std::tuple< unsigned, unsigned >, std::set< unsigned > > fractiongroup_label_to_sample;
 
+      if (msfile_section_[0].fraction != 1)
```

Comment:
add comment above:
// ensure that fraction group and fraction identifiers start with 1

Reply:
see other